modules/exploits/windows/misc/poisonivy_bof.rb

```diff
@@ -27,8 +27,8 @@ def initialize(info = {})
         [
           [ 'OSVDB', '83774' ],
           [ 'EDB', '19613' ],
-          [ 'URL', 'http://www.signal11.eu/en/research/articles/targeted_2010.pdf' ],
-          [ 'URL', 'http://samvartaka.github.io/malware/2015/09/07/poison-ivy-reliable-exploitation/' ],
+          [ 'URL', 'https://www.signal11.eu/en/research/articles/targeted_2010.pdf' ],
```

Comment:
the cert here is for home.pl, not signal11.eu.  I'm not sure using a wrong cert is necessarily more secure than using no cert.

Reply:
So if i revert this link back to `HTTP` it will be merged?